### PR TITLE
Code of Conduct

### PR DESCRIPTION
**Pull request by untitaker**
@geier @hobarrera

The commits are separated from the initial Covenant for easy diffing.

### CODE REVIEW

**Comment by hobarrera** [comment]
Maybe rebase 00a5c38cfd56f8df780698145500cbd5b5cff2a9 into e965349eb61f896e07510d933014f08c71241d45, but that's me just being super picky.

:+1: LGTM

**Comment by untitaker** [comment]
Fixed.

**Comment by hobarrera** [comment]
Yes, I mean different ones, sorry.
LGTM as-is though.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+``pimutils`` and ``vdirsyncer``, as well as the ``#pimutils`` IRC channel.
+
+It also applies when representing the core project on any other platform. In
+general, keep in mind that you are in public and that your actions determine
+the public perception of the project.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at [INSERT EMAIL ADDRESS]. All
```
Comment (untitaker) [inline CODE_OF_CONDUCT.md]: This needs to be fixed. Should we add our email addresses or should I create a `pimutils.org` address that relays?

**Comment by geier** [comment]
I'd prefer an @pimutils.org address.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (hobarrera) [inline CODE_OF_CONDUCT.md]: We can also just point to this issue tracker, if you don't feel comfortable with an email here.

**Comment by untitaker** [comment]
I've set up an email address `contact@pimutils.org`. It forwards to the email addresses found on your GitHub profiles. If you want to change that, please contact me.

**Comment by hobarrera** [comment]
That's fine by me. Do we get CC'd so we can reply-all?

**Comment by untitaker** [comment]
@hobarrera It looks like this:

    From: foo@bar
    Delivered-To: hugo@barrera.io
    To: contact@pimutils.org

If you reply-all, you will CC `contact@pimutils.org` which has the same effect anyway.

**Comment by hobarrera** [comment]
Great! :+1:

**Pull request merged**